docs/current/sdk/python/628797-get-started.md

```diff
@@ -34,9 +34,15 @@ The code samples in this tutorial are based on the above FastAPI project. If usi
 ## Step 1: Install the Dagger Python SDK
 
 :::note
-The Dagger Python SDK requires [Python 3.10 or later](https://docs.python.org/3/using/index.html). Using a [virtual environment](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments) is recommended.
+The Dagger Python SDK requires [Python 3.10 or later](https://docs.python.org/3/using/index.html). 
 :::
 
+Create a [virtual environment](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments) for your project (if you don't already have one):
```

Comment:
I would keep the "For example:" at the end. It's to clarify that there's other ways to do this. I don't even like putting `.venv` in the current dir and most people use some tool to manage this (e.g., virtualenv, poetry, conda...).

Reply:
Done